front/app/api/project_folders/useDeleteProjectFolder.ts

```diff
@@ -42,7 +42,7 @@ const useDeleteProjectFolder = () => {
         queryKey: adminPublicationsStatusCountsKeys.items(),
       });
       queryClient.invalidateQueries({
-        queryKey: homepageBuilderKeys.all(),
+        queryKey: contentBuilderKeys.all(),
```

Comment:
Could we invalidate only the content builder layouts for Folders?

Reply:
In this case we should invalidate all as the folder could be used on the homepage

Reply:
Ahh clear! Good point :+1: